roles/cloud_connector/templates/rhc-cloud-connector-worker.j2

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+CONFIG_FILE="{{ cloud_connector_config_file }}" /usr/libexec/yggdrasil-worker-forwarder
```

Comment:
This location depends upon -- https://github.com/theforeman/foreman-packaging/pull/7758